operator/cmd/mesh/profile-list_test.go

```diff
@@ -38,7 +38,7 @@ func TestProfileList(t *testing.T) {
 		t.Fatalf("failed to execute istioctl profile command: %v", err)
 	}
 	output := out.String()
-	expectedProfiles := []string{"default", "demo", "empty", "minimal", "openshift", "preview", "remote"}
+	expectedProfiles := []string{"default", "demo", "empty", "minimal", "openshift", "preview"}
```

Comment:
Actually, this looks like it's also missing the external profile, which should have been there already.
```suggestion
	expectedProfiles := []string{"default", "demo", "empty", "minimal", "openshift", "preview", "external"}
```

Reply:
done